files/socialmedia/update.json

```diff
@@ -0,0 +1,8 @@
+{
+	"packageManager": "github",
+	"name": "socialmedia",
+	"repo": "jabranr/socialmedia",
+	"files": {
+		"include": ["socialmedia.js", "socialmedia.min.js"]
```

Comment:
Hi @jimaek, I have added a `basePath` to `/dist` and removed non-minified JS as requested above. Cheers